src/celeritas/user/StepCollector.hh

```diff
@@ -57,11 +57,6 @@ class StepCollector
                   size_type max_streams,
                   ActionRegistry* action_registry);
 
-    // Default destructor and move
-    ~StepCollector();
-    StepCollector(StepCollector&&);
-    StepCollector& operator=(StepCollector&&);
```

Comment:
I think these need to be defaulted in the `.cc` file due to `unique_ptr` to a declared-but-undefined class. There are at least a couple places in the code where we do this.